### PR TITLE
Fix the normalization logic in seasonal risk

### PR DESCRIPTION
**Pull request by frozenhelium**
- Add option to include Lack of Coping capacity for Risk score
- Remove option to include Lack of Coping capacity for People exposed and People at Risk of Displacement
- Use current month as default value in country seasonal risk month selector

## Addresses:

- https://github.com/IFRCGo/go-web-app/issues/1545
- https://github.com/IFRCGo/go-web-app/issues/1230 (partially)

### CODE REVIEW

File: app/src/views/CountryProfileRiskWatch/MultiMonthSelectInput/index.tsx [hunk 1]
```diff
@@ -14,13 +14,7 @@ import type { SetValueArg } from '@togglecorp/toggle-form';
...
 import i18n from './i18n.json';
 import styles from './styles.module.css';
-
-const keyList = Array.from(Array(12).keys());
-const defaultValue = listToMap(
-    keyList,
-    (key) => key,
-    () => false,
-);
+import { monthKeyList, multiMonthSelectDefaultValue } from '#utils/constants';
```
Comment (barshathakuri) [inline app/src/views/CountryProfileRiskWatch/MultiMonthSelectInput/index.tsx]: Lint errors.

**Review by barshathakuri — APPROVED** [review]

File: app/src/components/domain/RiskSeasonalMap/index.tsx [hunk 2]
```diff
@@ -528,16 +528,24 @@ function RiskSeasonalMap(props: Props) {
...
                     (item) => item.population_in_thousands,
                 ),
             ) ?? 1;
+            */
+
+            const totalPopulation = sumSafe(
+                populationListSafe.map(
+                    (item) => item.population_in_thousands,
+                ),
+            ) ?? 1;
```
Comment (tnagorra) [inline app/src/components/domain/RiskSeasonalMap/index.tsx]: What if this sumSafe is zero? We do not have checks later about division by zero.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/src/views/CountryProfileRiskWatch/index.tsx [hunk 3]
```diff
@@ -38,7 +43,10 @@ export function Component() {
     const [
         selectedMonths,
         setSelectedMonths,
-    ] = useInputState<Record<number, boolean> | undefined>({ 0: true });
+    ] = useInputState<(typeof multiMonthSelectDefaultValue) | undefined>({
+        ...multiMonthSelectDefaultValue,
+        [getCurrentMonth()]: true,
+    });
```
Comment (tnagorra) [inline app/src/views/CountryProfileRiskWatch/index.tsx:49]: The default month is not selected. Is there any other requirement?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/src/components/domain/RiskSeasonalMap/index.tsx [hunk 4]
```diff
@@ -528,16 +528,24 @@ function RiskSeasonalMap(props: Props) {
...
 
             const populationFactorMap = listToMap(
                 populationListSafe,
                 (result) => result.iso3,
-                (result) => result.population_in_thousands / maxPopulation,
+                (result) => 1 - (result.population_in_thousands / totalPopulation) ** (1 / 4),
```
Comment (tnagorra) [inline app/src/components/domain/RiskSeasonalMap/index.tsx:552]: Do we have any reference for using this metric?

Comment (tnagorra) [reply app/src/components/domain/RiskSeasonalMap/index.tsx:552]: Added a NOTE regarding this metric.

**Review by tnagorra — APPROVED** [review]

File: app/src/views/CountryProfileRiskWatch/index.tsx (hunk 3 above)
Comment (tnagorra) [reply app/src/views/CountryProfileRiskWatch/index.tsx:49]: It is selected only after the data is fetched from the server.

NOTE: This change is for a country profile risk watch while the other change is for global risk watch.

**Pull request merged**